api/sda/sda.go

```diff
@@ -176,6 +177,27 @@ func Download(c *gin.Context) {
 		return
 	}
 
+	c.Header("Content-Length", fmt.Sprint(fileDetails.DecryptedSize))
```

Comment:
does s3 only have full file download, because of the full file size here in `Content-Length`?

Reply:
For now we only have full-file download! It seems that partial file access needs more changes to both the crypt4gh-library and the download service so that's gonna need to be a separate PR!